diff --git a/dist/leaflet.markercluster-src.js b/dist/leaflet.markercluster-src.js
index 7275d974e423c104ab7077ddd326722d8ba5da89..e2745278b1899b61a9fdcbf34625b7662f0b5beb 100755
--- a/dist/leaflet.markercluster-src.js
+++ b/dist/leaflet.markercluster-src.js
@@ -67,10 +67,10 @@
 				this.options.iconCreateFunction = this._defaultIconCreateFunction;
 			}
 
-			this._featureGroup = L.featureGroup();
+			this._featureGroup = new L.FeatureGroup();
 			this._featureGroup.addEventParent(this);
 
-			this._nonPointGroup = L.featureGroup();
+			this._nonPointGroup = new L.FeatureGroup();
 			this._nonPointGroup.addEventParent(this);
 
 			this._inZoomAnimation = 0;
@@ -89,7 +89,7 @@
 
 			// Hook the appropriate animation methods.
 			var animate = L.DomUtil.TRANSITION && this.options.animate;
-			L.extend(this, animate ? this._withAnimation : this._noAnimation);
+			Object.assign(this, animate ? this._withAnimation : this._noAnimation);
 			// Remember which MarkerCluster class to instantiate (animated or not).
 			this._markerCluster = animate ? L.MarkerCluster : L.MarkerClusterNonAnimated;
 		},
