diff --git a/L.Control.Geonames.js b/L.Control.Geonames.js
index ffc270208aa84f3f7fcced96091b0a44bd0ddab0..f36a5381fcbf11de0bc637a75a597e17d19012e7 100644
--- a/L.Control.Geonames.js
+++ b/L.Control.Geonames.js
@@ -130,7 +130,7 @@ L.Control.Geonames = L.Control.extend({
         }
 
         if (this.options.showMarker) {
-            this._marker = L.marker([lat, lon]).addTo(this._map);
+            this._marker = new L.Marker([lat, lon]).addTo(this._map);
 
             if (this.options.showPopup) {
                 this._marker.bindPopup(name);
@@ -140,7 +140,7 @@ L.Control.Geonames = L.Control.extend({
                 });
             }
         } else if (this.options.showPopup) {
-            this._popup = L.popup()
+            this._popup = new L.Popup()
                 .setLatLng([lat, lon])
                 .setContent(name)
                 .openOn(this._map)
@@ -363,7 +363,3 @@ L.Control.Geonames = L.Control.extend({
         return parts;
     }
 });
-
-L.control.geonames = function (options) {
-    return new L.Control.Geonames(options);
-};
\ No newline at end of file
