diff --git a/dist/L.Control.Locate.esm.js b/dist/L.Control.Locate.esm.js
index bb5a896e3a98e0b63a8444bd02e0150d4d8fe498..ddfb125a78fdbb644af9ccd10586ad195e5353f4 100644
--- a/dist/L.Control.Locate.esm.js
+++ b/dist/L.Control.Locate.esm.js
@@ -542,10 +542,10 @@ const LocateControl = Control.extend({
           padding: this.options.circlePadding,
           maxZoom: this.options.initialZoomLevel || this.options.locateOptions.maxZoom
         });
-        Util.requestAnimFrame(function () {
+        window.requestAnimationFrame(() => {
           // Wait until after the next animFrame because the flyTo can be async
           this._ignoreEvent = false;
-        }, this);
+        });
       }
     }
   },
@@ -677,7 +677,7 @@ const LocateControl = Control.extend({
       angle = Math.round(angle);
 
       this._compassHeading = angle;
-      Util.requestAnimFrame(this._drawCompass, this);
+      window.requestAnimationFrame(() => this._drawCompass());
     } else {
       this._compassHeading = null;
     }
@@ -881,9 +881,9 @@ const LocateControl = Control.extend({
    * Removes all classes from button.
    */
   _cleanClasses() {
-    DomUtil.removeClass(this._container, "requesting");
-    DomUtil.removeClass(this._container, "active");
-    DomUtil.removeClass(this._container, "following");
+    removeClasses(this._container, "requesting");
+    removeClasses(this._container, "active");
+    removeClasses(this._container, "following");
 
     removeClasses(this._icon, this.options.iconLoading);
     addClasses(this._icon, this.options.icon);
