diff --git a/dist/Leaflet.VectorGrid.bundled.js b/dist/Leaflet.VectorGrid.bundled.js
index eaf6520be79a64eb1fe227be81b5699d0bb5e1f1..b3bed408d66bb9d266c99c72ceb718f34e065c58 100644
--- a/dist/Leaflet.VectorGrid.bundled.js
+++ b/dist/Leaflet.VectorGrid.bundled.js
@@ -2055,7 +2055,7 @@ L.VectorGrid = L.GridLayer.extend({
 			if (this._map != null) {
 				renderer.addTo(this._map);
 			}
-			L.Util.requestAnimFrame(done.bind(coords, null, null));
+			window.requestAnimationFrame(done.bind(coords, null, null));
 		}.bind(this));
 
 		return renderer.getContainer();
