diff --git a/dist/L.Control.Locate.esm.js b/dist/L.Control.Locate.esm.js
index fd789051600197ae37e342aea432cce8d00944f6..789eaa9c8f6383c0010c3dcc79b792b1c27b5383 100644
--- a/dist/L.Control.Locate.esm.js
+++ b/dist/L.Control.Locate.esm.js
@@ -1,4 +1,4 @@
-import { Marker, setOptions, divIcon, Control, DomUtil, extend, LayerGroup, DomEvent, Util, circle } from 'leaflet';
+import { Marker, DivIcon, Control, DomUtil, LayerGroup, DomEvent, Util, Circle } from 'leaflet';
 
 /*!
 Copyright (c) 2016 Dominik Moritz
@@ -23,7 +23,7 @@ const removeClasses = (el, names) => {
  */
 const LocationMarker = Marker.extend({
   initialize(latlng, options) {
-    setOptions(this, options);
+    Util.setOptions(this, options);
     this._latlng = latlng;
     this.createIcon();
   },
@@ -47,7 +47,7 @@ const LocationMarker = Marker.extend({
 
     const icon = this._getIconSVG(opt, style);
 
-    this._locationIcon = divIcon({
+    this._locationIcon = new DivIcon({
       className: icon.className,
       html: icon.svg,
       iconSize: [icon.w, icon.h]
@@ -78,14 +78,14 @@ const LocationMarker = Marker.extend({
   },
 
   setStyle(style) {
-    setOptions(this, style);
+    Util.setOptions(this, style);
     this.createIcon();
   }
 });
 
 const CompassMarker = LocationMarker.extend({
   initialize(latlng, heading, options) {
-    setOptions(this, options);
+    Util.setOptions(this, options);
     this._latlng = latlng;
     this._heading = heading;
     this.createIcon();
@@ -332,16 +332,16 @@ const LocateControl = Control.extend({
     // set default options if nothing is set (merge one step deep)
     for (const i in options) {
       if (typeof this.options[i] === "object") {
-        extend(this.options[i], options[i]);
+        Object.assign(this.options[i], options[i]);
       } else {
         this.options[i] = options[i];
       }
     }
 
-    // extend the follow marker style and circle from the normal style
-    this.options.followMarkerStyle = extend({}, this.options.markerStyle, this.options.followMarkerStyle);
-    this.options.followCircleStyle = extend({}, this.options.circleStyle, this.options.followCircleStyle);
-    this.options.followCompassStyle = extend({}, this.options.compassStyle, this.options.followCompassStyle);
+    // Object.assign the follow marker style and circle from the normal style
+    this.options.followMarkerStyle = Object.assign({}, this.options.markerStyle, this.options.followMarkerStyle);
+    this.options.followCircleStyle = Object.assign({}, this.options.circleStyle, this.options.followCircleStyle);
+    this.options.followCompassStyle = Object.assign({}, this.options.compassStyle, this.options.followCompassStyle);
   },
 
   /**
@@ -361,16 +361,12 @@ const LocateControl = Control.extend({
     this._link = linkAndIcon.link;
     this._icon = linkAndIcon.icon;
 
-    DomEvent.on(
-      this._link,
-      "click",
-      function (ev) {
-        DomEvent.stopPropagation(ev);
-        DomEvent.preventDefault(ev);
-        this._onClick();
-      },
-      this
-    ).on(this._link, "dblclick", DomEvent.stopPropagation);
+    DomEvent.on(this._link, "click", (ev) => {
+      DomEvent.stopPropagation(ev);
+      DomEvent.preventDefault(ev);
+      this._onClick();
+    });
+    DomEvent.on(this._link, "dblclick", DomEvent.stopPropagation);
 
     this._resetVariables();
 
@@ -569,10 +565,10 @@ const LocateControl = Control.extend({
           padding: this.options.circlePadding,
           maxZoom: this.options.initialZoomLevel || this.options.locateOptions.maxZoom
         });
-        Util.requestAnimFrame(function () {
+        window.requestAnimationFrame(() => {
           // Wait until after the next animFrame because the flyTo can be async
           this._ignoreEvent = false;
-        }, this);
+        });
       }
     }
   },
@@ -625,7 +621,7 @@ const LocateControl = Control.extend({
       const style = this._isFollowing() ? this.options.followCircleStyle : this.options.circleStyle;
 
       if (!this._circle) {
-        this._circle = circle(latlng, radius, style).addTo(this._layer);
+        this._circle = new Circle(latlng, radius, style).addTo(this._layer);
       } else {
         this._circle.setLatLng(latlng).setRadius(radius).setStyle(style);
       }
@@ -704,7 +700,7 @@ const LocateControl = Control.extend({
       angle = Math.round(angle);
 
       this._compassHeading = angle;
-      Util.requestAnimFrame(this._drawCompass, this);
+      window.requestAnimationFrame(() => this._drawCompass());
     } else {
       this._compassHeading = null;
     }
@@ -908,9 +904,9 @@ const LocateControl = Control.extend({
    * Removes all classes from button.
    */
   _cleanClasses() {
-    DomUtil.removeClass(this._container, "requesting");
-    DomUtil.removeClass(this._container, "active");
-    DomUtil.removeClass(this._container, "following");
+    removeClasses(this._container, "requesting");
+    removeClasses(this._container, "active");
+    removeClasses(this._container, "following");
 
     removeClasses(this._icon, this.options.iconLoading);
     addClasses(this._icon, this.options.icon);
